katalog/cert-manager/README.md

```diff
@@ -1,34 +1,72 @@
 # cert-manager
+
 cert-manager is an automation tool for management and issuence of TLS
```

Comment:
there is a typo `issuance`. Is possible to take advantage of this PR and fix it?